tests/foreman/api/test_syncplan.py

```diff
@@ -193,11 +193,21 @@ def test_positive_create_with_interval(self):
         :CaseImportance: Critical
         """
         for interval in valid_sync_interval():
-            with self.subTest(interval):
+            if interval == SYNC_INTERVAL['custom']:
                 sync_plan = entities.SyncPlan(
-                    interval=interval,
+                    description=gen_string('alpha'),
                     organization=self.org,
+                    interval=interval,
+                    cron_expression=gen_choice((valid_cron_expressions()))
                 ).create()
+
+            else:
+                sync_plan = entities.SyncPlan(
+                     description=gen_string('alpha'),
+                     organization=self.org,
+                     interval=interval
+                ).create()
+
                 self.assertEqual(sync_plan.interval, interval)
```

Comment:
This assertion will run only for non-custom intervals. For custom interval, we create Sync Plan and assume that if it didn't throw error, everything is OK.
Is that intentional?

Reply:
No, that is a mistake. Thanks for noticing.